omod/src/test/java/org/openmrs/module/appointments/web/controller/AppointmentControllerTest.java

```diff
@@ -1,5 +1,6 @@
 package org.openmrs.module.appointments.web.controller;
```

Comment:
The changes in this file are not for this PR, but pending in another PR (see BAH-3222).  I need these in place to successfully build and test.